packages/framework/src/client/scheduler.js

```diff
@@ -79,91 +90,141 @@ export default class Scheduler {
 		return delay;
 	};
 
+	/**
+	 * Sets the data handlers for the scheduler.
+	 * @param {Function} dataRequested The dispatch-wrapped function for the data requested action.
+	 * @param {Function} dataReceived The dispatch-wrapped function for the data received action.
+	 */
+	setDataHandlers = ( dataRequested, dataReceived ) => {
+		this.dataRequested = dataRequested;
+		this.dataReceived = dataReceived;
+	};
+
 	/**
 	 * Finds a request that is either scheduled or overdue.
-	 * @param {string} resourceName The name of the resource to be read
-	 * @param {Date} now The current time
+	 * @param {string} resourceName The name of the resource for the operation
+	 * @param {string} operation The name of the operation to be performed (defaults to 'read')
```

Comment:
It doesn't look like a default of 'read' was set for the parameter `operation`. Same with `getInFlightRequests` below.

Reply:
You're correct, I removed the default as it wasn't really needed anymore. I'll update the jsdocs

Reply:
Fixed in 8ca82d44926d2e44786c0064deecb1ec248262f6